bin/brew

```diff
@@ -1,7 +1,10 @@
-#!/bin/sh
+#!/bin/bash
 
 chdir() {
-  cd "$@" >/dev/null
+  cd "$@" >/dev/null || {
+    echo "Error: failed to cd to " "$@" "!" >&2
```

Comment:
You'll probably need to escape the `!`, too. It's a special shell character for history expansion, and the expansion still happens inside double-quotes.